dashboard/test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -786,6 +786,49 @@ class OmniauthCallbacksControllerTest < ActionController::TestCase
     assert_equal auth[:credentials][:refresh_token], partial_user.oauth_refresh_token
   end
 
+  test 'google_oauth2: redirects to maker display code if present as user_return_to on successful google login' do
```

Comment:
Nit: I am having trouble parsing the title of this test. The comments in the tests are really helpful in understanding, but it would be great to see if we can make the title a little clearer.

Reply:
I tried to clarify the title, let me know if that helped :)

Reply:
Yes - thank you!